apps/settings/js/modules/bluetooth/bluetooth_device.js

```diff
@@ -28,11 +29,13 @@ define(function(require) {
    * @return {Observable} observableBluetoothDevice
    */
   return function ctor_bluetooth_device(device) {
+    var type = BtClassOfDeviceMapper.getDeviceType(device.cod);
     var observableBluetoothDevice = Observable({
       name: device.name,
       paired: device.paired,
       address: device.address,
-      cod: device.cod
+      cod: device.cod,
```

Comment:
I think It's better to keep the property which is given from platform. But I am not very insistent the decision here.